spec/models/form_profile_spec.rb

```diff
@@ -1951,6 +1951,8 @@ def expect_prefilled(form_id)
       it 'raises an IOError' do
         VCR.use_cassette('va_profile/disability/disability_rating_200_high_disability',
                          allow_playback_repeats: true) do
+          allow(FormProfile).to receive(:prefill_enabled_forms).and_return(['foo'])
+
           expect { described_class.new(form_id: 'foo', user:).prefill }.to raise_error(IOError)
```

Comment:
Can you write another test in this file that makes sure a form id that doesn't use prefill (your form number would be best) doesn't raise the IOError?